### DISCUSSION TITLE
Installation Help

### DISCUSSION THREAD

**Discussion by BananaBoy0924** [winstxnhdw/lc-hax#22] [Q&A] [answered]
I need help to install it, I don't know how to recursively clone the repository

**Comment by winstxnhdw** [comment]
I am not sure how I can help you, really.

**Comment by MichaelDavidGK** [comment, marked as answer]
> I need help to install it, I don't know how to recursively clone the repository

You put that command in cmd/powershell:
`git clone --recursive https://github.com/winstxnhdw/lc-hax.git`

You need to install git for this to work.
